xml/vk.xml

```diff
@@ -16684,7 +16684,7 @@ typedef void* <name>MTLSharedEvent_id</name>;
         </require>
     </feature>
 
-    <feature api="vulkansc" name="VKSC_VERSION_1_0" number="1.0" comment="Vulkan SC core API interface definitions">
+    <feature api="vulkansc" name="VKSC_VERSION_1_0" depends="VK_VERSION_1_2" comment="Vulkan SC core API interface definitions">
```

Comment:
should this be: 
```suggestion
    <feature api="vulkansc" name="VKSC_VERSION_1_0" depends="VKSC_VERSION_1_0" comment="Vulkan SC core API interface definitions">
```
?

Reply:
Definitely would not have it dependent on itself, that's at best a no-op and at worst an infinite loop. The 1.2 dependency is because SC 1.0 is defined as a derivative of Vulkan 1.2.

There are only two (intended) usages for this:

* Sort the features when emitting headers, in dependency order rather than 'number' order.
* Allow the build scripts to automatically incorporate the dependent versions rather than having to explicitly list them out. E.g. right now when you ask for 'makeSpec -version sc1.0' there's a static array of other things to automatically turn on:

>         'sc1.0' : [ 'VK_VERSION_1_0', 'VK_VERSION_1_1', 'VK_VERSION_1_2', 'VKSC_VERSION_1_0' ],

Reply:
I'm confused then, because VK_VERSION_1_1 .. VK_VERSION_1_3 depend on themselves in this MR. Are they supposed to depend on the previous version instead?

Reply:
Woops, yes. Thanks for catching that.